api/v1beta2/foundationdb_database_configuration.go

```diff
@@ -454,6 +510,28 @@ func (configuration DatabaseConfiguration) getRegionPriorities() map[string]int
 	return priorities
 }
 
+// AreSeparatedProxiesConfigured returns true if grv_proxies and
+// commit_proxies are greater than 0 (explicitly set) and Proxies is set
+// to 0
+func (configuration DatabaseConfiguration) AreSeparatedProxiesConfigured() bool {
+	counts := configuration.RoleCounts
+	if counts.Proxies == 0 && (counts.GrvProxies > 0 && counts.CommitProxies > 0) {
+		return true
+	}
+	return false
+}
+
+func (configuration DatabaseConfiguration) GetProxiesString(version Version) string {
```

Comment:
Missing the doc string `GetProxiesString ...`

Reply:
Done.